doc/tutorials/gaussian_transformation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>vaccum - &gt; vacuum</p><p><br></p><p>I would say :" to displace a vacuum mode in phase space"  (a vacuum mode is also a qumode)</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>